src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java

```diff
@@ -128,13 +134,9 @@ public DefaultLogger(OutputStream infoStream,
                          OutputStream errorStream,
                          OutputStreamOptions errorStreamOptions,
                          AuditEventFormatter messageFormatter) {
-        if (infoStreamOptions == null) {
-            throw new IllegalArgumentException("Parameter infoStreamOptions can not be null");
-        }
+        this.infoStreamOptions = infoStreamOptions;
+        this.errorStreamOptions = errorStreamOptions;
```

Comment:
we can safely remove this validation, as logic is null safe.
We will keep junit tests that pass nulls, to control code that it is still null safe.